graylog2-server/src/main/java/org/graylog/plugins/sidecar/migrations/V20180212165000_AddDefaultCollectors.java

```diff
@@ -104,13 +104,154 @@ public void upgrade() {
                         "   - name: Security"
         );
         ensureCollector(
-                "nxlog",
+                "nxlog-linux",
                 "exec",
                 "linux",
-                "/usr/lib/graylog-sidecar/nxlog",
+                "/usr/bin/nxlog",
                 "-f -c %s",
                 "-v -c %s",
-                ""
+                "define ROOT /usr/bin\n" +
+                        "\n" +
+                        "<Extension gelfExt>\n" +
+                        "  Module xm_gelf\n" +
+                        "  # Avoid truncation of the short_message field to 64 characters.\n" +
+                        "  ShortMessageLength 65536\n" +
+                        "</Extension>\n" +
+                        "\n" +
+                        "<Extension syslogExt>\n" +
+                        "  Module xm_syslog\n" +
+                        "</Extension>\n" +
+                        "\n" +
+                        "User nxlog\n" +
+                        "Group nxlog\n" +
+                        "\n" +
+                        "Moduledir /usr/lib/nxlog/modules\n" +
+                        "CacheDir /var/spool/collector-sidecar/nxlog\n" +
+                        "PidFile /var/run/graylog-sidecar/nxlog.pid\n" +
```

Comment:
Do we want to use Sidecar specific paths for collectors or maybe rely completely on default paths?

Reply:
I took these from the config generated by the old sidecar.
Never change a running system ;-)